gce-disk-expand/packaging/gce-disk-expand.spec

```diff
@@ -12,13 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 Name: gce-disk-expand
-Summary: Google Compute Engine root disk expansion utilities for EL6
-Version: 1.0.0
-Release: %(date +%s).el6
-License: GPLv3, Apache Software License
+Summary: Google Compute Engine root disk expansion module
+Version: 2.0.0
+Release: 1
+License: Apache Software License
 Group: System Environment/Base
 URL: https://github.com/GoogleCloudPlatform/compute-image-packages
-Requires: gawk, e2fsprogs, file, dracut, grep, util-linux, gdisk
+Source0: %{name}_%{version}.orig.tar.gz
+Requires: e2fsprogs, dracut, grep, util-linux, parted
 Conflicts: cloud-utils-growpart, cloud-utils, dracut-modules-growroot
```

Comment:
Ditto for these conflicts, keep the conflict for dracut-modules-growroot however.